adapters/platform_linux.c

```diff
@@ -75,6 +84,9 @@ STRING_HANDLE platform_get_platform_info(PLATFORM_INFO_OPTION options)
 
 void platform_deinit(void)
 {
+#ifndef DONT_USE_UPLOADTOBLOB
+    HTTPAPIEX_Deinit();
+#endif /*DONT_USE_UPLOADTOBLOB*/
```

Comment:
>* [](start = 8, length = 1)

nit: space

Reply:
all the places

---
In reply to: [319615293](https://github.com/Azure/azure-c-shared-utility/pull/366#discussion_r319615293) [](ancestors = 319615293)

Reply:
sure, tks! 

---
In reply to: [319615449](https://github.com/Azure/azure-c-shared-utility/pull/366#discussion_r319615449) [](ancestors = 319615449,319615293)